### ISSUE TITLE
[bug] File picker shows duplicate "Open" buttons with the quick preview open

### ISSUE THREAD

**Issue by alphabitserial** [pop-os/cosmic-files#542]
**Cosmic-files version:** 1.0.0.alpha.2.r20.g8dd26b6-1

**Issue/Bug description:** The file picker already has an "Open" button at the bottom of the window for confirming my selection, but right next to it is the quick preview pane's "Open" button. This creates unnecessary visual noise and potential confusion for less technically savvy users.

**Steps to reproduce:** Open the file picker dialog and click an image, opening the quick preview pane. For example, I have Firefox config key `widget.use-xdg-desktop-portal.file-picker` set to `1` to use COSMIC Files instead of the GTK picker.

**Expected behavior:** The quick preview panel should conditionally hide the "Open" button if the window is in a file picker context, to avoid needless duplication of functionality.

**Other notes:** See screenshot
![image](https://github.com/user-attachments/assets/5011f342-5723-437e-beb1-4cb6548179f7)

**Comment by iamkartiknayak** [comment]
isn't this a valid behavior? One `open` button at the bottom uploads the file where as other `open` button opens file in default app.

**Comment by alphabitserial** [comment]
Should the two labels be disambiguated? Maybe the Open button at the bottom could be labeled "Select" or "Confirm"

**Comment by iamkartiknayak** [comment]
Yeah, renaming the picker dialog button to select would be more appropriate.

**Comment by m1m1k4tz** [comment]
I’ve noticed this preview menu comes up when I click a file instead of popping up when I right click and choose it like I would expect it’s more noticeable on Asahi Linux

**Comment by alphabitserial** [comment]
I've updated the "expected behavior" portion of my original comment to reflect Kartik's comments :+1:
